packages/notifier/src/storesub.js

```diff
@@ -2,9 +2,76 @@
 import { E } from '@endo/eventual-send';
 import { Far, makeMarshal } from '@endo/marshal';
 import { observeIteration } from './asyncIterableAdaptor.js';
+import { makePublishKit, subscribeEach } from './publish-kit.js';
 import { makeSubscriptionKit } from './subscriber.js';
 
 /**
+ * Begin iterating the source, storing serialized iteration values.  If the
+ * storageNode's `setValue` operation rejects, the iteration will be terminated.
+ *
+ * Returns a StoredSubscriber that can be used by a client to directly follow
+ * the iteration themselves, or obtain information to subscribe to the stored
+ * data out-of-band.
+ *
+ * @template T
+ * @param {Subscriber<T>} subscriber
+ * @param {ERef<StorageNode>} storageNode
+ * @param {ERef<ReturnType<typeof makeMarshal>>} marshaller
```

Comment:
Why are only two of the three arguments allowed to be promises?
```suggestion
 * @param {ERef<Subscriber<T>>} subscriber
 * @param {ERef<StorageNode>} storageNode
 * @param {ERef<ReturnType<typeof makeMarshal>>} marshaller
```

Reply:
The reason `subscriber` is taken as a presence is so this can add methods to it to make the `StoredSubscriber`: https://github.com/Agoric/agoric-sdk/blob/b662e1995831973f996ccbfc823ccb4c7c896895/packages/notifier/src/storesub.js#L63-L69

Is there a way to do that on a remote ref?

In terms of value of making the change, I don't know of use cases where you'd get a remote subscriber and want to make a stored one out of it. It seems like that would be a different stream for which you'd want a repeater.

Reply:
I can think of some brittle approaches like `subscribeAfter: E(subscriber).subscribeAfter`, but I agree that the value probably isn't there. Perhaps we should just rename the other parameters to indicate the difference?
```suggestion
 * @param {Subscriber<T>} subscriber
 * @param {ERef<StorageNode>} storageNodeP
 * @param {ERef<ReturnType<typeof makeMarshal>>} marshallerP
```

Reply:
My understanding of the Agoric coding style is to actively avoid Hungarian notation. The types are there in the JSDoc and checked by tsc. There's no confusion with a presence to disambiguate.

Going to resolve this to leave as-is but feel free to reopen.